web-registry/src/components/molecules/BridgeInfo.tsx

```diff
@@ -0,0 +1,107 @@
+import { Box, Grid } from '@mui/material';
+
+import Card from 'web-components/lib/components/cards/Card';
+import SmallArrowIcon from 'web-components/lib/components/icons/SmallArrowIcon';
+import { Body, Label } from 'web-components/lib/components/typography';
+import { Title } from 'web-components/lib/components/typography/Title';
+import { Theme } from 'web-components/lib/theme/muiTheme';
+
+import { Link } from 'components/atoms';
+
+import bridgeMobile from 'assets/bridge.svg';
+import bridgeDesktop from 'assets/bridge-side.svg';
+
+const links = [
```

Comment:
Should we have this outside of the components?

Reply:
mmm.. good question. At the moment it seems like a pretty static and self-contained component. If it's okay, I'll keep it that way for now.